src/utils/Matching.ts

```diff
@@ -154,6 +154,34 @@ export class Matching {
     };
   }
 
+  public addPeerEventHandler({ handleDisconnected }: PeerEventHandler) {
+    if (this.peer === null) return;
+
+    const interval = setInterval(async () => {
+      if (this.peer === null) return;
+      const statistics = await this.peer.getStats(null);
+
+      statistics.forEach((report) => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        switch (report.type) {
+          case 'transport':
+            {
+              const transportReport = report as RTCTransportStats;
+
+              if (
+                transportReport.dtlsState === 'closed' ||
+                transportReport.dtlsState === 'failed'
+              ) {
+                handleDisconnected();
+                clearInterval(interval);
+              }
+            }
+            break;
+        }
```

Comment:
ask;
switch 문으로 작성해주셨는데, 추후 추가되는 case가 있는거죠?

p3;
switch문에 default 문 추가 부탁드릴게요!

```suggestion
        switch (report.type) {
          case 'transport':
          default:
            {
              const transportReport = report as RTCTransportStats;

              if (
                transportReport.dtlsState === 'closed' ||
                transportReport.dtlsState === 'failed'
              ) {
                handleDisconnected();
                clearInterval(interval);
              }
            }
            break;
        }
```

Reply:
넵! 아래 첨부드린 사진과 같이 다양한 타입이 존재하며, 차후 peer간 통신에 대한 데이터를 다룰 때를 대비하여 확장성있게 switch문을 사용하였습니다. (해당 PR이 머지되면 inbound-rtp 타입을 이용하여 사용자가 말하는 것을 관찰하는 로직을 추가할 예정입니다.)

<img width="215" alt="image" src="https://github.com/user-attachments/assets/b4b65644-f1d7-478b-a377-3ad33d7e0318">